src/server/src/entities/student.ts

```diff
@@ -6,7 +6,8 @@ import {
     JoinColumn,
     ManyToMany,
     JoinTable,
-    ManyToOne
+    ManyToOne,
+    TreeParent
```

Comment:
What is TreeParent for?

Reply:
That was something I was experimenting with because, for some reason, the resume and transcript entities were raising an error when they were declared as File datatypes.

Reply:
ohh okay